src/cli/test/test_cli.c

```diff
@@ -26,20 +26,13 @@
 #include <stddef.h>
 #include <setjmp.h>
 #include <cmocka.h>
-#include "cli/trn_cli.h"
-#include "trn_log.h"
 #include <stdio.h>
 #include <stdbool.h>
 #include <sys/resource.h>
 
```

Comment:
Droplet  tests would be helpful, but we can add those in a separate PR.

Reply:
Droplet test for loading/unloading is actually done as pre-task and post task for every other map test cases.